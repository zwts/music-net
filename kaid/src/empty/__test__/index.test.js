import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Empty from '../index';

describe('<Empty />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Empty>This page is empty, empty message here</Empty>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
