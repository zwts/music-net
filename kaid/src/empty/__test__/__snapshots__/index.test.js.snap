// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Empty /> renders correctly 1`] = `
<div
  className="kai-empty"
>
  <p
    className="kai-empty-text"
  />
</div>
`;
